static/regex/markdown/ipv6.md

```diff
@@ -1 +1,4 @@
-(don't ask me ¯\\_(ツ)_/¯)
\ No newline at end of file
+(don't ask me ¯\\_(ツ)_/¯)
```

Comment:
Why not remove it if you added the description? 🤔

Reply:
Done